app/logic/Calendar/ICal/ICalToEvent.ts

```diff
@@ -33,7 +34,31 @@ export function convertICalParserToEvent(ics: ICalParser, event: Event): boolean
     event.title = vevent.entries.summary[0].value;
   }
   if (vevent.entries.description) {
+    // Plaintext
     event.descriptionText = vevent.entries.description[0].value;
+    // HTML RFC 2445 4.2.1, 4.2, RFC 5545 3.2.1 and Thunderbird
+    // <https://datatracker.ietf.org/doc/html/rfc2445#section-4.2.1>
+    // <https://bugzilla.mozilla.org/show_bug.cgi?id=1607834>
+    let altrep = vevent.entries.description[0].properties.altrep;
```

Comment:
@NeilRashbrook BTW You got the nomenclature wrong. The DESCRIPTION is a "property", and the ALTREP is a "parameter" of the property.